lib/aws/session_store/dynamo_db/locking/base.rb

```diff
@@ -56,7 +56,7 @@ def delete_session(env, sid)
     def handle_error(env = nil, &block)
       begin
         yield
-      rescue AWS::DynamoDB::Errors::Base => e
+      rescue Aws::DynamoDB::Errors::Base => e
```

Comment:
@awood45 There was another reference to base class I needed to update, but specs are passing again.